internal/nozzle/nozzle.go

```diff
@@ -59,6 +90,7 @@ func NewNozzle(config *config.Config, tokenFetcher AuthTokenFetcher, log *gosten
 		parseAppMetricsEnable: config.AppMetrics,
 		stopper:               make(chan bool),
 		workersStopper:        make(chan bool),
+		messages: make(chan *loggregator_v2.Envelope, 10000),
```

Comment:
It's pretty much arbitrary. It's basically a buffer between the message producing loop and consuming workers, so I thought it might be a good idea to make it a bit larger 🤷‍♂

Reply:
Let's document that then.